cves/CVE-2013-0894.yml

```diff
@@ -348,7 +386,12 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The most notable mistake is that the developers copied an algorithm that
+    had a vulnerability and trusted it was fine. Another notable mistake was
```

Comment:
From what commits did you figure that the vulnerability came from the developers trusting each others' codes?